src/infill/SpaghettiInfillPathGenerator.cpp

```diff
@@ -1,28 +1,31 @@
-/** Copyright (C) 2017 Ultimaker - Released under terms of the AGPLv3 License */
+//Copyright (c) 2018 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
 #include "SpaghettiInfillPathGenerator.h"
 #include "../infill.h"
 #include "../FffGcodeWriter.h"
 
 namespace cura {
 
 
-bool SpaghettiInfillPathGenerator::processSpaghettiInfill(const SliceDataStorage& storage, const FffGcodeWriter& fff_gcode_writer, LayerPlan& gcode_layer, const SliceMeshStorage& mesh, const int extruder_nr, const PathConfigStorage::MeshPathConfigs& mesh_config, const SliceLayerPart& part, int infill_line_distance, int infill_overlap, int infill_angle, const Point& infill_origin)
+bool SpaghettiInfillPathGenerator::processSpaghettiInfill(const SliceDataStorage& storage, const FffGcodeWriter& fff_gcode_writer, LayerPlan& gcode_layer, const SliceMeshStorage& mesh, const size_t extruder_nr, const PathConfigStorage::MeshPathConfigs& mesh_config, const SliceLayerPart& part, int infill_line_distance, int infill_angle, const Point& infill_origin)
```

Comment:
Previously I didn't dare to touch these since I didn't want the g-code to change. Later I discovered why these were ints though: They were parsed from the settings in some cases as `std::vector<int>`. I then made a setting parser for `std::vector<AngleDegree>` and changed this one too.